src/lib/util/misc.c

```diff
@@ -36,6 +36,8 @@ RCSID("$Id$")
 #include <string.h>
 #include <sys/file.h>
 #include <sys/uio.h>
+#include <sys/types.h>
```

Comment:
Generally the include files are listed in alphabetical order.  So it should go sys/file.h sys/types.h and sys/uio.h